pkg/git/GitOperationService.go

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2020 Devtron Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package git
+
+import (
+	"fmt"
+	"github.com/devtron-labs/git-sensor/internal"
+	"github.com/devtron-labs/git-sensor/internal/sql"
+	"go.uber.org/zap"
+	_ "gopkg.in/robfig/cron.v3"
+	"gopkg.in/src-d/go-git.v4"
+)
+
+type GitOperationService interface {
+	FetchAndGetCommitInfo(pipelineMaterialId int, gitHash string) (*GitCommit, error)
```

Comment:
Fetch is same as Get, name should be FetchCommitInfo

Reply:
Not needed now. Have deleted the code